packages/chaire-lib-backend/src/utils/processManagers/TrRoutingProcessManager.ts

```diff
@@ -22,10 +29,17 @@ const startTrRoutingProcess = async (
     port: number,
     attemptRestart = false,
     threadCount = 1,
-    parameters: { debug?: boolean; cacheDirectoryPath?: string; cacheAllScenarios?: boolean } = {
-        debug: false,
-        cacheDirectoryPath: undefined,
-        cacheAllScenarios: false // Flag to enable the trRouting connection cache for all scenario
+    {
+        debug = false,
+        cacheDirectoryPath,
+        // Flag to enable the trRouting connection cache for all scenario
```

Comment:
Is it possible to keep the comment after the param to make it clear to which entry it's related.